content/next-docs/tutorials/istio-csr/istio-csr.md

```diff
@@ -136,6 +136,14 @@ istioctl install -f istio-install-config.yaml
 # istioctl install --set profile=openshift -f istio-install-config.yaml
 ```
 
+You will be prompted for input to confirm your choice of Istio profile:
```

Comment:
note: This is just #1022 but copied into next-docs - I didn't spot it when I reviewed that PR, so I'm fixing it here :grin: